src/neuro_api_tony/view.py

```diff
@@ -1655,26 +1656,32 @@ def on_create(self, event: wx.CommandEvent) -> None:
         """Handle create command event."""
         event.Skip()
 
+        ##application_config_folder = get_tony_application_config_folder()
+        ##if not application_config_folder.exists():
+        ##    application_config_folder.mkdir(parents=True)
+
         with wx.FileDialog(
             self,
             "Create Config File",
             wildcard="JSON files (*.json)|*.json|All files (*.*)|*.*",
             style=wx.FD_SAVE | wx.FD_OVERWRITE_PROMPT,
-            defaultDir=os.getcwd(),
-            defaultFile="tony-config.json",
+            defaultDir=str(Path.cwd()),  # TODO: Use application_config_folder
+            defaultFile="tony_config.json",
```

Comment:
Why the change from `tony-config.json` to `tony_config.json`? I feel like this just comes down to personal preference.
```suggestion
            defaultFile="tony-config.json",
```